pyproject.toml

```diff
@@ -37,11 +37,11 @@ mock = "^4.0.3"
 scandir = "^1.10.0"
 redis = "^4.3.5"
 jsonschema = "^4.17.1"
-pydantic = "^1.10.2"
+pydantic = "^1.10.18"
 PyDispatcher = "^2.0.6"
 pytz = "^2022.6"
 tzlocal = "^4.2"
-mxcubecore = ">=1.145.0"
+mxcubecore = ">=1.153.0"
```

Comment:
I guess this needs the version that will be released after the corresponding change in mxcubecore is merged, right?

Reply:
Yes, but the tests were failing with the older version of `mxcubecore` because of an update to the `AbstractBeam` interface. That is also why all other PR's having failing tests (I think)

Reply:
Ah!?

This? <https://github.com/mxcube/mxcubeweb/actions/runs/10904178365/job/30259973161#step:8:65>

```none
 >       assert ap == actual_aperture
E       AssertionError: assert 'A100' == 'slits'
E         - slits
E         + A100
```

Should be addressed separately: <https://github.com/mxcube/mxcubeweb/issues/1401>.

Reply:
This is an intermittent issue. I just run tests on latest mxcubeweb with mxcubecore 1.145.0 locally. All tests passed.

Reply:
I should add I did manage to make it fail as well, with the above assertion, after running the test many times.

Reply:
The problem comes from the evaluation of the size of the slits. If you have slits, smaller than the aperture, than the size is taken from the slits and the test_aperture-set fails systematically with 'A100' == 'slits'.  There are three solutions in order the tst to pass. Either: 
 1. Remove the slits from the beam.xml
 or
2. Set the slits bigger (or at least equal) of the biggest aperture (A100). The values are hardcoded in  the SlitsMockup, so we need to change the mockup to take into account the values, set in the configuration.
or
3. Remove the biggest aperture from the tests.

While waiting for the SlitMockup to be changed and to make it take the values from the configuration file, I vote for 1.

Reply:
Feels like it would make sense to discuss further on the dedicated ticket: https://github.com/mxcube/mxcubeweb/issues/1401

Reply:
> This is an intermittent issue. I just run tests on latest mxcubeweb with mxcubecore 1.145.0 locally. All tests passed.

The problem is intermittent, because the test are using random aperture size, so when A100 is not used, the tests pass